book-1-orientation/chapters/CLASSES_INTRO.md

```diff
@@ -281,6 +298,62 @@ namespace Classes
 }
 ```
 
+## Fields
+
+Much like properties, Fields are use to store values in an object. Unlike properties fields are usually private and cannot be accessed outside the object. The concept of storing private data that is only accessible inside the object is referred to as "encapsulation".
```

Comment:
Much like `properties`, `fields` are used to store values in an object.  Whereas properties are generally used to expose public state, fields are usually private and are used to store internal, or private, state.  This concept of storing private data that can only be accessed from inside the object is known as `encapsulation`.

Reply:
👍

Reply:
I'd rather make those words **bold** instead. I think using the `code` highlight suggests that those are keywords.

Reply:
that makes a lot of sense.  I'm for bold.  I just use ticks as my go to text differentiator cause i'm lazy.